src/envoy/http/authn/http_filter_factory.cc

```diff
@@ -23,19 +23,24 @@ namespace Envoy {
 namespace Server {
 namespace Configuration {
 
+namespace {
+// The name for the mixer server cluster.
+const std::string kAuthnFactoryName("istio_authn");
+}  // namespace
+
 class AuthnFilterConfig : public NamedHttpFilterConfigFactory,
                           public Logger::Loggable<Logger::Id::http> {
  public:
   HttpFilterFactoryCb createFilterFactory(const Json::Object& config,
                                           const std::string&,
-                                          FactoryContext& context) override {
+                                          FactoryContext&) override {
     ENVOY_LOG(debug, "Called AuthnFilterConfig : {}", __func__);
 
     google::protobuf::util::Status status =
         Utils::ParseJsonMessage(config.asJsonString(), &policy_);
     ENVOY_LOG(debug, "Called AuthnFilterConfig : Utils::ParseJsonMessage()");
     if (status.ok()) {
-      return createFilter(policy_, context);
+      return createFilter(policy_);
     } else {
       ENVOY_LOG(debug, "Utils::ParseJsonMessage() return value is NOT ok!");
```

Comment:
Use error or critical. May be even better to use PANIC macro instead.

Reply:
Changed to use PANIC.